doc/classes/Image.xml

```diff
@@ -333,6 +333,24 @@
 				Loads an image from the binary contents of a PNG file.
 			</description>
 		</method>
+		<method name="load_svg_from_buffer">
+			<return type="int" enum="Error" />
+			<param index="0" name="buffer" type="PackedByteArray" />
+			<param index="1" name="scale" type="float" default="1.0" />
+			<description>
+				Loads an image from the binary contents of a SVG file.
```

Comment:
Does this accept `.svgz` file contents? If so, this should be documented. Also, the "binary contents" should refer to UTF-8 data for uncompressed SVGs (`.svg`).

Reply:
I'll check about the `.svgz` and update the documentation, also about the "binary contents".

Thanks.

Reply:
@Calinou 

Looks like the **thorvg** does not support `.svgz` directly, but It's simply the case of detecting the **gzip** in the `buffer` and uncompress it before sending to **thorvg**.

I think it's a nice addition, I'll implement it and document as covered by the method. 👍🏼

Reply:
I thought better of it and "decompression" is out of "loading" scope, it's better to just warn about it and the user can pick whatever compression method he desires.